.gitignore

```diff
@@ -759,3 +759,6 @@ $RECYCLE.BIN/
 # .pnp.*
 
 # End of https://www.toptal.com/developers/gitignore/api/node,yarn,intellij,pycharm,webstorm,python,django,macos,windows,linux
+
+# Emacs backups
+*~
```

Comment:
Why was this added?

Reply:
Added to prevent emacs backup files that are generated automatically from edits to a file. 
- If I were working on a `src/schema.py` file and using emacs, it would generate a `src/schema.py~` on auto-save to prevent data loss.

Reply:
ooh okay. I visually missed the '.' before the ~. Thanks. This would avoid any temp files to be added as well.